.travis.yml

```diff
@@ -10,10 +10,11 @@ dist: xenial
 
 language: python
 python:
-  - "3.6"
   - "3.7"
   - "3.8"
   - "3.9"
+  - "3.10"
```

Comment:
Ubuntu 16.04 seems to be too old and doesn't have the newer python wheels?
We might want to just use azure here anyway, since we moved away from travis in other repos. See https://github.com/nexB/skeleton/commit/1165f12eb321cf3ecced9655424d6a376933c23c for reference

Reply:
Thanks for this deprecating travis.